lcli/src/generate_bootnode_enr.rs

```diff
@@ -15,6 +15,7 @@ pub fn run<E: EthSpec>(matches: &ArgMatches, spec: &ChainSpec) -> Result<(), Str
     let udp_port: NonZeroU16 = clap_utils::parse_required(matches, "udp-port")?;
     let tcp_port: NonZeroU16 = clap_utils::parse_required(matches, "tcp-port")?;
     let output_dir: PathBuf = clap_utils::parse_required(matches, "output-dir")?;
+    // FIXME: why is this being read from.. somewhere rather than just using the spec?
```

Comment:
We aren't actually using the spec from `testnet-dir` for `generate-bootnode-enr` at the moment.

The `spec` that we have access to here is the default mainnet/minimal/gnosis spec and shouldn't be used for all cases:
https://github.com/sigp/lighthouse/blob/ce8d0814ad71819b54c878f8c8a610c48e7eca40/lcli/src/main.rs#L636-L644

This closure loads the network config, and we should use this to derive the correct `chain_spec`
https://github.com/sigp/lighthouse/blob/ce8d0814ad71819b54c878f8c8a610c48e7eca40/lcli/src/main.rs#L695-L708